src/m_misc.c

```diff
@@ -3210,42 +3132,17 @@ boolean M_ParseOption(const char *p, boolean wad)
             else if (c == '+')
               c = '=';
 
-            if (!M_InputAddKey(dp->ident, c))
-              break;
+            M_InputAddKey(dp->input_id, c);
           }
           else
           {
-            int value = M_GetKeyForName(buffer);
-            if (value > 0)
-            {
-              if (!M_InputAddKey(dp->ident, value))
-                break;
-            }
-            else
-            {
-              value = M_GetJoyBForName(buffer);
-              if (value >= 0)
-              {
-                if (!M_InputAddJoyB(dp->ident, value))
-                  break;
-              }
-              else
-              {
-                value = M_GetMouseBForName(buffer);
-                if (value >= 0)
-                {
-                  // Don't bind movement and turning to mouse wheel. It needs to
-                  // be impossible to input a one-frame of movement
-                  // automatically in speedrunning.
-                  if (M_IsMouseWheel(value) &&
```

Comment:
I think it was a DSDA requirement to disallow binding movement actions to mouse wheels?

Reply:
Binding is allowed, but the player should not use it in strict mode. I will fix this later in another place.